### DISCUSSION TITLE
LAMMPS install in linux cerver with cpu

### DISCUSSION THREAD

**Discussion by kingji-cms** [mir-group/nequip#368] [Q&A]
Hello. 
I want to install LAMMPS for nequip in my own server without gpu (only cpu).

![image](https://github.com/mir-group/nequip/assets/97725018/b8589c02-d8f9-4f2a-a227-1cede90e2796)

The manual for lammps installation is working in Google colab.

However, when I tried to install own linux centos7 server, I have some error messages.

whe I typed below command, I have this error...
cmake …/cmake -DCMAKE_PREFIX_PATH=python -c 'import torch;print(torch.utils.cmake_prefix_path)' && make -j4"

![image](https://github.com/mir-group/nequip/assets/97725018/d1b2e77d-344e-4ff3-85d0-d20da5248602)

I tried to other command such as 
cmake ../cmake -DCMAKE_PREFIX_PATH=`python -c 'import torch;print(torch.utils.cmake_prefix_path)'` -DCMAKE_CXX_FLAGS="-DUSE_CPU" && make -j4

but, result is not change..

I'm trying to install anaconda virtual environment.
I already install torch in my virtual environment with cpu version. (conda install pytorch torchvision torchaudio cpuonly -c pytorch)

How can I install LAMMPS for nequip in my own server.

Thank you.
JiHoon Kim.

**Comment by Linux-cpp-lisp** [comment]
Are you using PyTorch 1.11?

**Reply by kingji-cms** [reply]
Oh, I'm using pytorch 2.0.1... I'm sorry for asking a foolish question..
Now, I',m retrying pytorch 1.10.2 cpu version.

However, I have other error messages like below image.
![image](https://github.com/mir-group/nequip/assets/97725018/358e93ce-82df-4794-812e-f0abae5b40e9)

I'm still trying to install anaconda virtual environment.

Thank you.
